package.json

```diff
@@ -54,6 +55,7 @@
   "dependencies": {
     "dotenv": "^8.2.0",
     "express": "^4.17.1",
-    "postinstall": "^0.7.0"
+    "postinstall": "^0.7.0",
```

Comment:
Not related to this Pull Request, but  this module should not be required.

Reply:
Ok, but I think, this might have everyone from your template..😄